# Second Order Feedback Linearization 6-DOF ANTSMC Controller for the VTec U-IV gains

alpha: [1.8, 1.8, 1.8, 1, 1, 1]
beta_: [1.5, 1.5, 1.5, 1.5, 1.4, 1.4]
K2: [0.8, 0.8, 0.3, 1, 1, 1]
K_alpha: [0.1, 0.1, 0.1, 2.2, 0.3, 1.7]
mu: [0.001, 0.001, 0.001, 0.001, 0.001, 0.001]
K_min: [0.01, 0.01, 0.01, 6, 1.2, 7]
K1_init: [1.4, 0.0, 0.0, 0.0, 0.0, 0.0]

# alpha: [1.8, 1.8, 1.8, 1, 1, 1]
# beta_: [1.5, 1.5, 1.5, 1.5, 1.4, 1.4]
# K2: [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
# K_alpha: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1]
# mu: [0.001, 0.001, 0.001, 0.001, 0.001, 0.001]
# K_min: [0.01, 0.01, 0.01, 0.01, 0.01, 0.01]
# K1_init: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]